network-api/networkapi/wagtailpages/pagemodels/buyersguide/editorial_content_index.py

```diff
@@ -39,7 +39,7 @@ class BuyersGuideEditorialContentIndexPage(
     def get_context(self, request, *args, **kwargs):
         context = super().get_context(request, *args, **kwargs)
         context["home_page"] = self.get_parent().specific
-        context["items"] = self.get_descendants().public().live()
+        context["items"] = self.get_descendants().specific().public().live()
```

Comment:
**Suggestion**

Probably not a large queryset but is it more efficient to call specific after the `live` and `public` filtering is done?

Reply:
Ah good call. Yea, it's not going to be that big in the beginning but it will grow. So good to make this as performant as we can now.

Reply:
Updated in 54694c4a639213f08f69bde206b3e0c3a3e8bc33